### PR TITLE
Fix tests on Julia v0.7

### PR DESCRIPTION
**Pull request by kmsquire**
* Parsing seems to have changed slightly in Julia (or JuliaParser?) between v0.6 and v0.7.
  For test/data/testparser.jl, line 10 is the end of a @doc string (`""" ->`), and Line 11
  is an expression (`f6(x) = 6x`)
* In v0.6, the zero count goes to line 10, and in v0.7, the zero count goes (more correctly?)
  to line 11

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -46,10 +46,17 @@ cd(dirname(@__DIR__)) do
...
-    # See https://github.com/JuliaLang/julia/issues/9663 (when this is fixed, can uncomment the next line on julia 0.4)
-    target = Union{Int64,Void}[nothing, 1, nothing, 0, nothing, 0, nothing, nothing, nothing, 0, nothing, nothing, nothing, nothing, nothing, nothing, 0, nothing, nothing, 0, nothing, nothing, nothing, nothing]
-    #target = Union{Int64,Void}[nothing, 1, nothing, 0, nothing, 0, nothing, nothing, nothing, nothing, nothing, nothing, nothing, nothing, nothing, nothing, 0, nothing, nothing, 0, nothing, nothing, nothing, nothing]
+
+    # Parsing seems to have changed slightly in Julia (or JuliaParser?) between v0.6 and v0.7.
+    # Line 10 is the end of a @doc string (`""" ->`), and Line 11 is an expression (`f6(x) = 6x`)
+    # In v0.6, the zero count goes to line 10, and in v0.7, the zero count goes (more correctly?)
+    # to line 11
+
+    if VERSION < v"0.7.0-"
```
Comment (tkelman) [inline test/runtests.jl]: any idea more specifically what caused the change? looks like it happened between 2 and 3 months ago, judging by travis logs - https://travis-ci.org/JuliaCI/Coverage.jl/builds/234799535 vs https://travis-ci.org/JuliaCI/Coverage.jl/builds/248594841

Comment (kmsquire) [reply test/runtests.jl]: I don't have time to test or do a proper `git bisect` right now, but https://github.com/JuliaLang/julia/pull/21746/commits is one possiblilty.

Comment (kmsquire) [reply test/runtests.jl]: But we'll probably need to do a git bisect to find out for sure.

My real interest here is to attempt to discover why coverage is dropping significantly for most projects (such as the `codecov/project` below--see #132).  I'm not even sure this is related, so might want to do a `git bisect` for this as well, if possible.

Comment (kmsquire) [reply test/runtests.jl]: Running `git bisect` now.

Comment (kmsquire) [reply test/runtests.jl]: `git bisect` says the culprit is "[various improvements to parser #22161](https://github.com/JuliaLang/julia/pull/22161)", which makes much more sense than my initial guess.

I've updated the pull request to reference this specific commit.  Not sure right now if it has anything to do with #132, although it's probably worth testing.

Comment (kmsquire) [reply test/runtests.jl]: Bump

**Pull request merged**